### PR TITLE
Add xml.preferences.showSchemaDocumentationType preference

### PR DESCRIPTION
**Pull request by xorye**
Signed-off-by: David Kwon <dakwon@redhat.com>

### CODE REVIEW

**Comment by xorye** [comment]
The settings have been updated:
settings gif

File: package.json [hunk 1]
```diff
@@ -231,6 +231,24 @@
...
+          "type": "string",
+          "default": "all",
+          "enum": [
+            "documentation",
+            "appinfo",
+            "all",
+            "none"
+          ],
+          "markdownEnumDescriptions": [
+            "Documentation comes from the `documentation` element.",
```
Comment (angelozerr) [inline package.json:244]: I will add `xs:` for each element like` xs:documentation`

**Comment by angelozerr** [comment]
@xorye please fix my comment and after that I'm OK to merge it.

File: package.json (hunk 1 above)
Comment (xorye) [reply package.json]: Thanks, fixed

**Pull request merged**